modules/users/client/config/users.client.routes.js

```diff
@@ -192,25 +220,28 @@
       }).
       state('profile.accommodation', {
         url: '/accommodation',
-        templateUrl: '/modules/users/views/profile/profile-view-accommodation.client.view.html',
+        templateUrl: profileviewAccommodationTemplateUrl,
         requiresAuth: true,
         noScrollingTop: true,
         data: {
           pageTitle: 'Profile accommodation',
         },
       }).
+      /*
+      @TODO find out what is wrong with this, template path did not seem to exist
```

Comment:
The template `modules/offers/client/views/offer-host-view.client.view.html` does not exist and the URL does not show anything on production either (e.g. https://www.trustroots.org/profile/mikael/meet), but now the templates are explicitly imported we can't get away with invalid templates paths.

Any idea what that is about? @simison / @mrkvon?

Reply:
`git log -- modules/offers/client/views/offer-host-view.client.view.html` shows this was once present, but got moved to a directive.

Reply:
Ah, when the _meet_ type was added, it looks like there was originally going to be a page on the profile showing the users _meets_ and this `/profile/<username>/meet` route would have been the mobile page to show that, but even in the original incarnation the template url was just the same as for the _host_ type (`offer-host-view.client.view.html`), so wasn't implemented correctly. 

See https://github.com/Trustroots/trustroots/blob/afad74c0a5a3a7d91c9b851d671878f39dacf6d5/modules/users/client/config/users.client.routes.js#L206-L214

Conclusion: I should just remove this route.

Reply:
Sounds about right. :-)